src/Encoder/RailFenceCipher.js

```diff
@@ -0,0 +1,173 @@
+
+import Encoder from '../Encoder'
+
+const meta = {
+  name: 'rail-fence-cipher',
+  title: 'RailFence cipher',
```

Comment:
I think we should stick to the same title typo as on the [Wikipedia article](https://en.wikipedia.org/wiki/Rail_fence_cipher): Rail fence cipher